### PR TITLE
[16.0][IMP] web_help: Improve helpers

### PR DESCRIPTION
**Pull request by tarteo**
Add some helper functions from version 15.0

### CODE REVIEW

File: web_help/static/src/helpers.esm.js [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
         return matchedTrips[0].Trip;
     }
     return null;
 }
+
+export function wait(ms) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export async function waitUntilAvailable(selector, ms = 50) {
```
Comment (Tardo) [inline web_help/static/src/helpers.esm.js:24]: A bit "hard" to traverse the DOM tree every 50ms by default. Perhaps you could use MutationObserver, but the implementation would be more complex and probably not worth the effort.

**Review by Tardo — APPROVED** [review]
A small visual glitch with the icon. It can be fixed by moving the icon inside '.o_cp_bottom_right':
icon_help

**Comment by tarteo** [comment]
> A small visual glitch with the icon. It can be fixed by moving the icon inside '.o_cp_bottom_right': icon_help

@Tardo Thanks fixed it

**Pull request merged**